mergify_engine/queue/merge_train.py

```diff
@@ -280,6 +286,44 @@ async def create_pull(self) -> None:
 
         await self.update_summaries(check_api.Conclusion.PENDING)
 
+    def _generate_merge_queue_pull_body(
+        self,
+        queue_rule: typing.Optional[rules.EvaluatedQueueRule] = None,
+    ) -> str:
+        description = f"""
+{self._get_embarked_refs(markdown=True)} are embarked together for merge.
+
+This pull request has been created by Mergify to speculatively check the mergeability of #{self.user_pull_request_number}.
+You don't need to do anything, Mergify will close it automatically when your check-runs have finished.
```

Comment:
```suggestion
You don't need to do anything. Mergify will close this pull request automatically when it is complete.
```

I don't think "check-runs have finished" is the only case where it's closed, so rather be vague?